Packs/CrowdStrikeFalconStreamingV2/Integrations/CrowdStrikeFalconStreamingV2/CrowdStrikeFalconStreamingV2.py

```diff
@@ -393,27 +393,25 @@ async def long_running_loop(
                 demisto.createIncidents(incident)
                 offset_to_store = int(event_offset) + 1
                 if last_integration_context_set + timedelta(minutes=1) <= datetime.utcnow():
-                    integration_context = demisto.getIntegrationContext()
+                    integration_context = get_integration_context()
                     integration_context['offset'] = offset_to_store
                     if store_samples:
                         try:
                             sample_events_to_store.append(event)
                             demisto.debug(f'Storing new {len(sample_events_to_store)} sample events')
                             sample_events = deque(json.loads(integration_context.get('sample_events', '[]')), maxlen=20)
                             sample_events += sample_events_to_store
-                            integration_context['sample_events'] = json.dumps(list(sample_events))
+                            integration_context['sample_events'] = list(sample_events)
```

Comment:
Consider merging `smaple_events` using `object_keys` (if they have a unique id).

Reply:
it doesnt really matter if there will be duplicates there, it is just for sampling events